system/trace/trace_dump.c

```diff
@@ -223,28 +223,45 @@ static void copy_task_name(FAR char *dst, FAR const char *src)
 FAR static struct trace_dump_task_context_s *get_task_context(pid_t pid,
                                       FAR struct trace_dump_context_s *ctx)
 {
-  FAR struct trace_dump_task_context_s **tctxp;
-  tctxp = &ctx->task;
-  while (*tctxp != NULL)
+  FAR struct trace_dump_task_context_s *tctxp;
```

Comment:
I don't see the benefit to change tctxp from AR struct trace_dump_task_context_s ** to AR struct trace_dump_task_context_s *.